README.md

```diff
@@ -62,61 +65,63 @@ Finally, two sample .NET apps interact with the resources deployed to showcase t
+> [! WARNING]
+> [Azure Key Vault in this sample has network rules to deny public internet access, but has an exception to allow the current user's IP address for this quickstart setup. In real environments we recommend removing this rule and integrating your service access through [Private Endpoint](https://learn.microsoft.com/en-us/azure/key-vault/general/private-link-service?tabs=portal).]
 
-* From the root directory of this repo, run the first part of the deployment. The script will use the environment variables and build composed resource names by appending the `PREFIX` variable as Azure resource names.
+1. From the root directory of this repo, run the first part of the deployment. The script will use the environment variables and build composed resource names by appending the `PREFIX` variable as Azure resource names.
 
 ```bash
 ./deploy/quickstart-agw-storage.sh
 ```
 
-* It will take a few minutes to deploy all resources. Keep the terminal open and take note of some of the generated DNS entries.
-* This scripts also uploads a `sample.txt` file to a Blob storage container and generates a SAS URI token for testing.
-* From a bash terminal, try out a `CURL` command to the DNS of the Public IP address attached to the Application Gateway.
+2. It will take a few minutes to deploy all resources. Keep the terminal open and take note of some of the generated DNS entries.
+3. This scripts also uploads a `sample.txt` file to a Blob storage container and generates a SAS URI token for testing.
+4. From a bash terminal, try out a `CURL` command to the DNS of the Public IP address attached to the Application Gateway.
 
 ```bash
 curl --insecure "<APP_GATEWAY_SASURI copied output>"
 ```
 
-* Run the same command with the Blob Public URI printed out by the script `BLOB_SASURI` value. Verify this does not succeed since this Storage account is blocking direct Internet traffic.
+5. Run the same command with the Blob Public URI printed out by the script `BLOB_SASURI` value. Verify this does not succeed since this Storage account is blocking direct Internet traffic.
+6. Configure your custom DNS (through your DNS provider) and SSL certificate for end-to-end SSL encryption.
 
-* Configure your custom DNS and SSL certificate for end to end SSL encryption.
+   1. Ensure you create a `CNAME` or `A` Record pointing to the DNS of the Azure Public IP created above.
+   2. `CNAME` is the simpler approach and you can point it to the value output of the script in the form of `xxx.<region>.cloudapp.azure.com`.
+   3. Create an SSL certificate for this domain, or ensure you have a valid wildcard domain.
+   4. Export the PFX file into a base64 encoded file. For example `base64 -w 0 ./temp/<YOURFILENAME_LOCATION>.pfx > ./temp/<YOURFILENAME_LOCATION>.pfx.base64`.
+   5. Update the Key vault secret:
```

Comment:
~~~suggestion
   1. Ensure you create a `CNAME` or `A` Record pointing to the DNS of the Azure Public IP created above.
   
      `CNAME` is the simpler approach and you can point it to the value output of the script in the form of `xxx.<region>.cloudapp.azure.com`.

   1. Create an SSL certificate for this domain, or ensure you have a valid wildcard domain.

   1. Export the PFX file into a base64 encoded file.

      For example.
      
       ```bash
       base64 -w 0 ./temp/<YOURFILENAME_LOCATION>.pfx > ./temp/<YOURFILENAME_LOCATION>.pfx.base64
      ```

1. Update the Key Vault secret.
~~~